create3_lidar/launch/slam_toolbox_launch.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# Copyright 2022 iRobot Corporation. All Rights Reserved.
+
+from launch import LaunchContext
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+
+
+def generate_launch_description():
```

Comment:
any reason to use this launch file rather than the original one https://github.com/SteveMacenski/slam_toolbox/blob/ros2/launch/online_async_launch.py ?